.github/ISSUE_TEMPLATE/1-bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Report a bug you found
```

Comment:
Given this is a mono-repo. We can add a field for project or package name. e.g.: create-plugin, sign-plugin and other upcoming tools.

Reply:
I've migrated to using github forms for this issue template.